docker/service/node/run

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh -eu
+# If this is deployed as an offline signer, skip running the node
+# so that we don't waste time attempting to connect.
+if [ -n "$OASIS_ROSETTA_GATEWAY_OFFLINE_MODE" ]; then
```

Comment:
Due to `set -u` we probably need to use something like:

```suggestion
if [ "${OASIS_ROSETTA_GATEWAY_OFFLINE_MODE:-no}" != "no" ]; then
```

Reply:
the gateway itself checks empty/non-empty. `${...:-}` probably

Reply:
```suggestion
if [ -n "$OASIS_ROSETTA_GATEWAY_OFFLINE_MODE-" ]; then
```

Reply:
Would that work?

Because I tried something similar in the terminal and it complained:
```sh
$ set -u
$ [ -n "$OASIS_ROSETTA_GATEWAY_OFFLINE_MODE-" ] && echo "yes" || echo "no"
bash: OASIS_ROSETTA_GATEWAY_OFFLINE_MODE: unbound variable
```

Reply:
oh my bad, forgot the curly braces